### ISSUE TITLE
Sometimes unable to view diff on test failure

### ISSUE THREAD

**Issue by roberttaylor426** [cursive-ide/cursive#2723]
Hi,

The docs suggest that it should be possible to view the reason for a test assertion failing by hovering over the error line:

![image](https://user-images.githubusercontent.com/535032/183892174-42321e86-dbfd-48dd-aadd-2ebc93787a0d.png)

and then view a diff:

![image](https://user-images.githubusercontent.com/535032/183892313-e22d11eb-34cf-44e7-97f7-b47fff2d6889.png)

In previous versions this used to work as advertised, but recently I don't predictably see details of test errors. For example in this `testing` block titled 'some behaviour' I see:

![no-test-error-diff](https://user-images.githubusercontent.com/535032/183903600-7c0842e5-ee30-4b3a-9bd8-6aff59344748.gif)

There's no option to compare the diff, and the error message is of no help understanding what the issue is.

I note that if I compare two strings then I _do_ get the option to view the diff by hitting `Alt-Shift-Enter`:

![test-error-diff](https://user-images.githubusercontent.com/535032/183903921-9ed954d4-3e51-42a0-a9b1-18609edaec31.gif)

In short there are some unexpected values that can be diffed, and some that can't, and I'd love to be able to see an 'Expected' / 'Actual' for _all_ test errors. What am I doing wrong please?

Cursive version: `1.12.4-2022.1`
Intellij version: `IC-221.5787.3`

**Comment by jtrunick** [comment]
This feature was working for me great, but then just stopped working, and I have the same behavior as described above. I'm also on the same Cursive version: 1.12.4-2022.1, maybe I was auto-updated to a non-working version, not sure.

**Comment by guess-burger** [comment]
I too used this feature a lot. Unfortunately working with tests without this context does make things more difficult than they need to be (see examples below). It have since rolled back to an older version of IntelliJ + Cursive to get this feature back as it does appear to be an issue with `1.12.4` (possible with the addition of https://github.com/cursive-ide/cursive/issues/2387 ?).

Intellij: `2022.2.3`
Cursive: `1.12.4-2022.2`
![image](https://user-images.githubusercontent.com/8173463/195326428-a68d17b5-a260-427e-9d9a-a8f747290c0c.png)

Intellij: `2021.3.3`
Cursive: `1.12.3-2021.3`
![image](https://user-images.githubusercontent.com/8173463/195326593-bbd73d16-201e-4ef7-b8db-7eb54e1b8cbb.png)

**Comment by guess-burger** [comment]
I can confirm this was also working with Intellij: `2022.1.4`, Cursive: `1.12.3-2022.1`

**Comment by alinposho** [comment]
+1 to getting this fixed. I'm experiencing the behaviour describe above with Cursive version `1.12.4-2022.2` (Personal licence).

**Comment by mustajarvi** [comment]
@cursive-ide, would it be possible to create a 1.12.3-2022.2(.4) release which could be used in the meantime? Not being able to see the diff has a substantial workflow impact (at least for me). Thanks!
